classes/helper.php

```diff
@@ -69,5 +79,28 @@ public static function reason_display(int $reason): string {
                 return (string) $reason;
         }
     }
-}
 
+    /**
+     * Returns a formatted time duration in m:s.ms format.
+     * @param float $duration
+     * @return string
+     * @throws \Exception
+     */
+    public static function duration_display(float $duration): string {
+        $ms = round($duration * 1000, 0);
+        $s = round($duration, 0);
+        $di = new \DateInterval('PT' . $s . 'S');
+        return $di->format('%i:%S.') . $ms;
+    }
+
+    /**
+     * Returns HTTP status as a badge.
+     *
+     * @param int $status
+     * @return string
+     */
+    public static function http_status_display(int $status): string {
+        $spanclass = 'badge ' . self::STATUS_BADGES[$status / 100];
```

Comment:
I think this is breaking on a CLI script which has a null status

I think in the case of cli scripts we can do the same stuff, ie 0 = success and non zero is error. I'm not sure if we are actually storing cli return values or if that's actually possible until the script exists so just make this work

Reply:
Can you also most this column to the far left before the request